yarn.config.cjs

```diff
@@ -38,10 +38,16 @@ module.exports = defineConfig({
       const isChildWorkspace = workspace.cwd !== '.';
       const isPrivate =
         'private' in workspace.manifest && workspace.manifest.private === true;
+      const isDocs = workspace.cwd.startsWith('docs/');
       const dependenciesByIdentAndType = getDependenciesByIdentAndType(
         Yarn.dependencies({ workspace }),
       );
 
+      // Completely ignore docs workspaces
+      if (isDocs) {
+        continue;
+      }
```

Comment:
This ignores all the checks like a package must be called `@metamask`, all packages must have a version, description, homepage, LICENSE,etc